pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -44,10 +46,10 @@ var populationStrategy = map[string]CertificateCreationCallback{
 		}
 		keyPair, _ := triple.NewServerKeyPair(
 			caKeyPair,
-			VirtOperatorServiceName+"."+secret.Namespace+".pod.cluster.local",
+			VirtOperatorServiceName+"."+secret.Namespace+"."+caPodClusterLocal,
```

Comment:
then here instead do `fmt.Spring(localPodDNSTemplateString, VirtOperatorServiceName, secret.Namespace)` 

I think that reads nicer but :shrug:

Reply:
Thanks, it does look better now